src/parser/shared/modules/features/BaseHealerStatValues.js

```diff
@@ -416,6 +419,7 @@ class BaseHealerStatValues extends Analyzer {
   moreInformationLink = null;
   statistic() {
     const results = this._prepareResults();
+    this.url = '';
```

Comment:
I think this should be a local variable instead of a class variable, since it's only used here.

Reply:
So I wasn't too sure on the different between local and class variables. I think its just the naming convention, but if I am wrong please let me know. I just pushed the update that I believe covers this point.

Thanks.

Reply:
I meant that it should be a variable scoped within the method rather than as a class property, so it's not used anywhere else.

Reply:
Gotcha. Hopefully good to go now.